components/script/dom/webidls/Navigator.webidl

```diff
@@ -28,7 +28,7 @@ interface NavigatorID {
 
 // https://webbluetoothcg.github.io/web-bluetooth/#navigator-extensions
 partial interface Navigator {
-  readonly attribute Bluetooth bluetooth;
+  [SameObject, Pref="dom.bluetooth.enabled"] readonly attribute Bluetooth bluetooth;
```

Comment:
@nox you are right. I forgot to mention, I added the `dom.bluetooth.enabled` pref here. Without this pref here, other Bluetooth interfaces can be accessed via this attribute without passing `--pref="dom.bluetooth.enabled"` when running servo, even though those interfaces have this restriction also and should not be accessed. As far as I know the purpose of this `dom.bluetooth.enabled` pref (which is  set to `false`) is to disable the webbluetooth features by default, but without this addition, it does not.